connector/src/main/java/org/geysermc/connector/utils/LoginEncryptionUtils.java

```diff
@@ -201,13 +213,26 @@ public static boolean authenticateFromForm(GeyserSession session, GeyserConnecto
                 } else if (formId == AUTH_FORM_ID && window instanceof SimpleFormWindow) {
                     SimpleFormResponse response = (SimpleFormResponse) window.getResponse();
                     if (response != null) {
-                        if (response.getClickedButtonId() == 0) {
-                            showLoginDetailsWindow(session);
-                        } else if (response.getClickedButtonId() == 1) {
-                            session.disconnect("Login is required");
+                    	if(GeyserConnector.getInstance().getConfig().getRemote().getAuthType() == "smart"){
```

Comment:
Add whitespace (`if (GeyserConnector.getInstance().getConfig().getRemote().getAuthType() == "smart") {`) to match with the rest of the Geyser codebase.